internal/config/config.go

```diff
@@ -49,6 +58,33 @@ type peer struct {
 	Password      string         `yaml:"password"`
 }
 
+type peerAutodiscovery struct {
```

Comment:
Should it be peerAutoDiscovery ? Here and in the next occurrences

Reply:
Personally I wouldn't write "auto discovery" in free text, therefore I wouldn't separate the word "autodiscovery" using caps here, either. To me "autodiscovery" makes more sense but I'm not too strongly opinionated about that.

Similar examples:

- Automount
- AutoRun (a single word despite the caps)

Reply:
Ack, thanks